conjur/errors.py

```diff
@@ -25,4 +25,13 @@ def __init__(self, message="Error: Failed to run command to completion."):
         super().__init__(self.message)
 
 class MissingRequiredParameterException(Exception):
-    """ Exception for when user does not input a required paramter """
+    """ Exception for when user does not input a required parameter """
```

Comment:
@sigalsax is this a remark or a message? or???

Reply:
this is a comment (: